js/initcontext.go

```diff
@@ -153,6 +153,11 @@ func (i *InitContext) requireModule(name string) (goja.Value, error) {
 func (i *InitContext) requireFile(name string) (goja.Value, error) {
 	// Resolve the file path, push the target directory as pwd to make relative imports work.
 	pwd := i.pwd
+	if filepath.IsAbs(name) {
+		i.logger.Warnf("you import '%s' with an absolute path, this won't be cross platform and likely will not work if"+
+			" you move the script between machines. Also it might not work on k6 cloud if it doesn't have `file://` in front",
```

Comment:
```suggestion
		i.logger.Warnf("you import '%s' with an absolute path, this won't be cross platform and likely will not work if"+
			" you move the script between machines or in cloud. You should use `file://` in front to have the full cross platform and cloud finally supported",
```
I would make "the suggestion" more explicit

Reply:
The thing is that "full cross platform" isn't true, as just moving the script between machines will still not work. It will work if you archive it and run the archive, but not if you than unarchive it on a different machine.

Reply:
The reality of it is that the correct way for something to be cross-platform and cross-machine (as the OS is not the only problem) is to *not* use absolute paths at all.

So I really do prefer if that is the sentiment we sent more than "this won't work in the cloud if you are on Windows and use absolute paths without file://"

cc @na-- @imiric